Misc/SimpleCalculator.java

```diff
@@ -0,0 +1,262 @@
+import java.util.*;
+import java.lang.*;
+import java.awt.*;
+import java.awt.event.*;
+import javax.swing.*;
+
+public class SimpleCalculator extends JFrame implements ActionListener
+{
+	JFrame jf; //object for the frame
+	JTextField screen; //object for calculator screen
+	float newValue=0,newValue2; //store values in the screen before and after oprtator is clicked
+	int flag=0; //to identify the operator
+	JButton plus,clear,close,minus,div,mul,equal;
+	JButton seven,eight,nine,four,five,six,one,two,three,zero,dot;
+	
+	public SimpleCalculator()
+	{
+		jf=new JFrame();                                       /*this is code to create a frame*/
+		jf.setTitle("Calculator");
+		jf.setLayout(null);
+		jf.setSize(600,700);
+		jf.setVisible(true);
```

Comment:
good to move at the end of ctor

Reply:
sorry @ayaankhan98  i didn't get your point.
could you please elaborate the same.
and thanks for helping.